lib/constants.dart

```diff
@@ -1,7 +1,15 @@
-const ALARMS = 'alarms';
-const ALARM_SETS = 'alarm_sets';
-const FOCUS_DURATION_KEY = 'focus_duration';
-const BREAK_DURATION_KEY = 'break_duration';
-const FROM_TIME_OF_DAY_KEY = 'from_date';
-const TO_TIME_OF_DAY_KEY = 'to_date';
-const ALARM_DATABASE = 'alarm_database';
+/// Database constants
```

Comment:
nit: Zameni ///komentare //komentarima (// vs ///) da bi se tretirali kao komentari za sekcije ili opste komentare (//), a ne kao dokumentacija za varijablu ispod (///). Takodje i ispod i na drugim mestima ako znas da ima.
```suggestion
// Database constants
```

Reply:
Siii